ci/jenkins/test.sh

```diff
@@ -375,10 +375,96 @@ function revert_snapshot_windows {
     sleep 5
 }
 
+function deliver_antrea_linux {
```

Comment:
This function was used in docker job and it has been removed in https://github.com/antrea-io/antrea/pull/6019/files#diff-378579753493ec9807a7826d7f5941124aad0fa32f9b174d462f02915cc3ef3d